### ISSUE TITLE
Systemd-ukify  fails since update

### ISSUE THREAD

**Issue by edpil02** [systemd/systemd#30582]
### systemd version the issue has been seen with

255

### Used distribution

Fedora Rawhide

### Linux kernel version used

kernel-6.7.0-0.rc6.48.fc40.x86_64

### CPU architectures issue was seen on

None

### Component

systemd

### Expected behaviour you didn't see

I use systemd-boot and systemd-ukify to boot  my machine , but systemd-ukify fails since update : 

Assuming obsolete command line syntax with no verb. Please use 'build'.
Traceback (most recent call last):
  File "/usr/bin/ukify", line 1668, in <module>
    main()
  File "/usr/bin/ukify", line 1654, in main
    opts = parse_args()
           ^^^^^^^^^^^^
  File "/usr/bin/ukify", line 1649, in parse_args
    finalize_options(opts)
  File "/usr/bin/ukify", line 1630, in finalize_options
    raise ValueError('--output= must be specified when building a PE addon')
ValueError: --output= must be specified when building a PE addon

Here are my settings :  
 
/usr/lib/systemd/ukify build  \
--linux=/lib/modules/$(uname -r)/vmlinuz  \
--initrd=/boot/efi/$(cat /etc/machine-id)/$(uname -r)/initrd   \
--cmdline="$(cat /etc/kernel/cmdline)"   \
--splash=./logos/fedora-logo2.bmp \
--output=/boot/efi/EFI/Linux/linux-$(uname -r).efi

### Steps to reproduce the problem

always

**Comment by mrc0mmand** [comment]
I can't seem to reproduce this. Could you show how you invoke `ukify` together with the respective output verbatim, i.e. without any additional formatting involved?

**Comment by edpil02** [comment]
I just tried my script again and it seems to work again without understanding why.
But since I have other weird problems with rawhide that can be explained.
Thanks.

$ :   sudo  ./ukify 
Splash image logos/fedora-logo2.bmp is 728×410 pixels
Kernel version not specified, starting autodetection 😖.
Found uname version: 6.7.0-0.rc7.54.fc40.x86_64
Wrote unsigned /boot/efi/EFI/Linux/linux-6.7.0-0.rc7.54.fc40.x86_64.efi
